setup/.lintstagedrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
-  'setup/!(*test).{js,ts,json}': [
+  '!(*test).{js,ts,json}': [
     'eslint --cache --fix',
-    () => 'ncc build',
-    () => 'git add dist'
+    () => 'npm run bundle',
+    () => 'git add setup/dist/ setup/lib/'
   ],
-  'setup/src/**/*.ts': () => 'tsc -p tsconfig.json',
-  'setup/*.{js,ts,json,md}': 'prettier --write'
+  'src/**/*.ts': () => 'tsc -p tsconfig.json',
+  '*.{js,ts,json,md}': 'prettier --write'
 };
```

Comment:
I had to revert these, since it wasn't detecting the staged files, because `.lintstagedrc.js` resides inside the `setup` folder.

Reply:
That's fine. I tinkered a bit with getting the lintstaged file working again and if it's still not quite working, I have a few other things I can try.